kpops/cli/main.py

```diff
@@ -38,7 +38,7 @@
 )
 
 DEFAULT_PATH_OPTION: Path = typer.Option(
-    default=Path("defaults"),
+    default=None,
```

Comment:
`None` can't be set as a default for `Path` as they are not compatible types

Reply:
Oh, I didn't know. Strangely it worked as intended when I tried `kpops generate` and all the tests passed. Does that mean that the tests' coverage is not good?

Reply:
Python's type hints are only meant for static analysis and ignored at runtime. Please change `Path` to `Optional[Path]` and update all the references below. I suppose mypy doesn't catch this mistake because it doesn't have knowledge on how typer works

Reply:
Any idea how I can make a Path() object falsy?

Reply:
Ah I hadn't refreshed the page and didnt see[ your response](https://github.com/bakdata/kpops/pull/151#discussion_r1135667375)